lib/byteLengthForComponentType.js

```diff
@@ -25,6 +28,7 @@ function byteLengthForComponentType(componentType) {
         case WebGLConstants.UNSIGNED_SHORT:
             return 2;
         case WebGLConstants.FLOAT:
+        case WebGLConstants.UNSIGNED_INT:
```

Comment:
Also update `readBufferComponentType` and `writeBufferComponentType` with the new type.